src/TritonRoute/src/dr/FlexDR_init.cpp

```diff
@@ -845,9 +858,9 @@ void FlexDRWorker::initNet_termGenAp_new(const frDesign* design, drPin* dPin)
                             instPinRectBBox.right(),
                             instPinRectBBox.top());
           if (!boost::polygon::intersect(pinRect, routeRect)) {
-            if (instPinRectBBox.distMaxXY(routeBox) <= halfWidth)
-              outOfRouteBoxRects.push_back(
-                  make_pair(instPinRectBBox, currLayerNum));
+//            if (instPinRectBBox.distMaxXY(routeBox) <= halfWidth)
+//              outOfRouteBoxRects.push_back(
+//                  make_pair(instPinRectBBox, currLayerNum));
```

Comment:
remove rather than comment, here and throughout.

Reply:
I think all of these comments are from another branch merged to master, because they are not my doing. I will remove them now.

Reply:
Nobody else is changing dr besides you at the moment.  If they are from another branch they would be still be from you.  If you think otherwise then you should check into the origin.